profiles/sink/Sink_VideoDevice.yaml

```diff
@@ -36,7 +36,5 @@ dsVideoDevice:
       #dsVIDEO_CODEC_MAX         = (0x01 << 3)    /*!< Out of range */
       SupportedVideoCodingFormats: 0x06  #dsVIDEO_CODEC_MPEG2|dsVIDEO_CODEC_MPEG4PART10
       SupportedDisplayFramerate:
-        - "60"
-        - "50"
-        - "24"
+        - "3840x2160px60"
```

Comment:
Please check this description and update the supported frame rates 
https://github.com/rdkcentral/rdk-halif-device_settings/blob/develop/include/dsVideoDevice.h#L409

Reply:
Supported FrameRates for 4K platforms:

3840x2160px47
3840x2160px48
3840x2160px50
3840x2160px59
3840x2160px60

Supported FrameRates for 2K:
1366X768px47
1366X768px48
1366X768px50
1366X768px59
1366X768px60

Reply:
updated